examples/advanced-project/src/__checks__/uptime/url.check.ts

```diff
@@ -0,0 +1,53 @@
+import { UrlAssertionBuilder, UrlMonitor } from 'checkly/constructs'
+import { uptimeGroup } from '../utils/website-groups.check' 
+
+
+// URL Monitors are the simplest and most efficient uptime monitors, they only
+// check for the status code of the response.
+// Further documentation: https://www.checklyhq.com/docs/url-monitors/
+
+new UrlMonitor('homepage-url-check', {
+  name: 'Homepage URL Monitor',
+  activated: true,
+  group: uptimeGroup,
+  maxResponseTime: 10000, // milliseconds
+  degradedResponseTime: 5000,
+  request: {
+    url: 'https://www.danube-web.shop/',
+    followRedirects: true,
+    assertions: [
+      UrlAssertionBuilder.statusCode().equals(200),
+    ]
+  }
+})
+
+// In this extension example, we create multiple URL monitors at once
+
+// Here we're using a short array of URLS, but this example can be extended
+// by parsing a sitemap or JSON file and adding every URL to an array.
+const sitemapUrls = [
+    'https://danube-web.shop/books/2',
```

Comment:
Total nit: but can we not change indentation from 2 to 4 spaces here? Prettier is of help if you're not using it. :)

Reply:
yup, late edit not caught by prettier, should look right now.